agentlib/utils/multi_agent_system.py

```diff
@@ -91,11 +91,15 @@ def add_agent_logger(config: AgentConfig, sampling=60) -> AgentConfig:
             sampling=
         """
         # Add Logger config
+        filename = f"variable_logs//Agent_{config.id}_Logger.log"
         cfg = {
             "module_id": "AgentLogger",
             "type": "AgentLogger",
             "t_sample": sampling,
             "values_only": True,
+            "filename": filename,
+            "overwrite_log": True,
+            "clean_up": False,
```

Comment:
While at it, maybe specify `variable_logging_setting` as a field to pass clean_up, overwrite_log, and a filename folder? Maybe `variable_logs` is not a good name for others.